cat-loving-app/src/services/breedsService.ts

```diff
@@ -0,0 +1,67 @@
+export interface Breed {
+    id: string;
+    name: string;
+    description: string;
+    reference_image_id: string;
+}
+
+export interface BreedDetails {
+    name: string;
+    description: string;
+    origin: string;
+    temperament: string;
+}
+
+const getAllBreeds = async ():Promise<Breed[]> => {
+    try {
+        const response = await fetch('https://api.thecatapi.com/v1/breeds', {
+            headers: {
+                'x-api-key': 'live_Ka5tWZF7c0Gm8UhINCls6XSG8wmnHHVyxF8gFnQcnXXl4zPTA8RZcAWSsnpCphEm'
```

Comment:
Why did you choose to use this api key this way?

Reply:
Yes. It was not in the requirement any detail on the authentication of that or how to retrieve the key. If we had a single key that was represented by our service I would put it as an env variable that I could set on my deployment pipeline. This variable can also be a secret ( like it is possible with GithubActions )